portfolio/src/main/webapp/gallery-script.js

```diff
@@ -18,24 +16,42 @@ const addProjectsToDOM = async (json) => {
     json will always contain a top-level object which has a single member.
     this member will be an array of objects. these objects will have three properties:
     - title : a string containing the title of the project
-    - URL : a string which contains a URL linking to the blog post about
+    - URL : a string which contains a URL linking to the source code of
         the project
     - preview : a string which contains a URL linking to the preview image
         for this project
+    - description : a string which contains a short description of this project
+    - blog_link : a string which contains a URL linking to a blog post about
+        the project
```

Comment:
Since json is passed into the function as a parameter this should be documented above the function right?

Reply:
I've improved this now 👍 Let me know if you think it needs more work https://github.com/thomasogara/step-repository/pull/4/commits/4fb6f19068b90f3e5c7344050fbd8fb2ff620008